### ISSUE TITLE
both scripts giving ssl error

### ISSUE THREAD

**Issue by birdybro** [MAME-GETTER/MiSTer_MAME_SCRIPTS#28]
This has been happening since the most recent linux update.

```
/media/fat/scripts# update_hbmame-getter.sh
STARTING: HBMAME-GETTER

Downloading the most resent hbmame-merged-set-getter script.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) error:1411C146:SSL routines:tls1_prf:unsupported digest type
/media/fat/scripts# update_mame-getter.sh
STARTING: MAME-GETTER

Downloading the most recent mame-merged-set-getter script.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) error:1411C146:SSL routines:tls1_prf:unsupported digest type
```

https://curl.se/libcurl/c/libcurl-errors.html

```
CURLE_SSL_CONNECT_ERROR (35)

A problem occurred somewhere in the SSL/TLS handshake. You really want the error buffer and read the message there as it pinpoints the problem slightly more. Could be certificates (file formats, paths, permissions), passwords, and others. 
```

Not sure why this is happening. Has archive.org changed the method of handling the digest auth? Or maybe the new mister linux update changed the version of openssl and curl? Just some thoughts.

One side note:

In general `--path-as-is` should be used for scripts like this, as you may have 1-2 `.`'s which can lead to squashing of the URL (however there may be contingencies that make this inappropriate). I thought this was what was happening at first, but probably not upon second look. This might actually have to do with the ssl settings on the mister. This was a fresh install of the mister on my microSD card btw, after running update (which updated linux).

**Comment by MAME-GETTER** [comment]
I'm wondering if some hosts at archive.org have changed their minimal ciphers?

**Comment by birdybro** [comment]
btw, hbmame 0221, 0222, sfz.hack, link in the script is down on archive, should probably be removed.

Gulunpa also doesn't download (the script worked with updater-pc when i was testing it, since not running on the mister itself, that's going to be my current workaround), I had to manually go find it.  It's not found in the current links. 0.227-0.229 are all up on archive.org waiting to be added ;)

**Comment by MAME-GETTER** [comment]
@birdybro I have removed the 0221 0222 hbmame sets from the script as they are no longer working on AO. I have also removed sfz.hack from the script as it longer on AO too. :-( 

I have been told that MRA for sfz.hack will be removed from the JT github soon as well so there is no need for this anymore. 

As for your SSL Issues. I cant duplicate the issue. I suspect this is an issue with the mirrored host for github that you were connected to for downloading the script. Maybe get hub was doing a cert or proxy maintenance on that host. Let me know if the issue continues. 

Closing
